packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -154,12 +162,9 @@ enum SdkBuiltInKindsMiscellaneousEnum {
   any = "any",
 }
 
-export type SdkBuiltInKinds =
-  | keyof typeof SdkBuiltInKindsMiscellaneousEnum
-  | keyof typeof SdkIntKindsEnum
-  | keyof typeof SdkFloatKindsEnum
-  | keyof typeof SdkGenericBuiltInStringKindsEnum
-  | keyof typeof SdkAzureBuiltInStringKindsMapping;
+export type SdkBuiltInKinds = Exclude<IntrinsicScalarName, SdkBuiltInKindsExcludes> | "any";
+
+export type SdkBuiltInKindsExcludes = "utcDateTime" | "offsetDateTime" | "duration";
```

Comment:
so I think this should not be exported. I will make a change to remove the export.